fitbenchmarking/fitbenchmarking
# tag gradient_free optimizers as global

#### Description of Work

Fixes #1018 


#### Testing Instructions

1. Check the gradient_free minimizers show up in the list of global optimizers
2.
3.

Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?
